packages/@aws-cdk/aws-glue/lib/table.ts

```diff
@@ -30,20 +30,23 @@ export enum TableEncryption {
   SSE_S3 = 'SSE-S3',
 
   /**
-   * Server-side encryption (SSE) with an AWS KMS customer managed key.
+   * Server-side encryption (SSE) with an AWS KMS key managed by KMS.
    *
    * @see https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingKMSEncryption.html
    */
   SSE_KMS = 'SSE-KMS',
 
   /**
-   * Client-side encryption (CSE) with an AWS KMS customer managed key.
+   * Server-side encryption (SSE) with an AWS KMS key managed by the account owner.
+   */
+  SSE_KMS_MANAGED = 'SSE-KMS-MANAGED',
+
+  /**
+   * Client-side encryption (CSE) with an AWS KMS key managed by the account owner.
    *
    * @see https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingClientSideEncryption.html
-   *
-   * TODO: implement. It's not clear what properties to set on a table to support client-side encryption.
    */
-  // CSE_KMS = 'CSE-KMS'
+  CSE_KMS = 'CSE-KMS'
```

Comment:
Proposed names that pass the jsii naming bar:

- `Unencrypted`
- `SSE_KMS` => `Kms`
- `SSE_KMS_MANAGED` => `KmsManaged`
- `SSE-S3` => `S3Managed`
- `CSE_KMS` => `ClientKms`

Reply:
Your way distinguishes CSE with the prefix `Client` and implies SSE for the others. It's efficient. I think I'd prefer `ClientSideKms` over `ClientKms`, though.

Reply:
Sounds good to me.

I am okay with `ServerSideXxx` as well, but then we'll have to also change it in other places ;-) and I favor consistency at this point.